app/api/routes.py

```diff
@@ -307,12 +338,18 @@ def create_resource(json, db):
     try:
         db.session.add(new_resource)
         db.session.commit()
+        index.save_object(new_resource.serialize_algolia_search)
+
+    except AlgoliaUnreachableHostException as e:
```

Comment:
Annoyingly, `AlgoliaUnreachableHostException` does not extend from `AlgoliaException`, so we have to catch both. Probably can use the same logic you have in there now for the AlgoliaException too